tools/sdk-generation-pipeline/packages/sdk-generation-lib/src/lib/publishResult.ts

```diff
@@ -0,0 +1,241 @@
+import * as child_process from 'child_process';
+import * as fs from 'fs';
+import * as path from 'path';
+import { Connection, createConnection } from 'typeorm';
+
+import { getGenerateAndBuildOutput } from '../types/taskInputAndOuputSchemaTypes/GenerateAndBuildOutput';
+import { AzureBlobClient } from '../utils/blob/AzureBlobClient';
+import { requireJsonc } from '../utils/requireJsonc';
+import { logger } from '../utils/logger';
+import { AzureSDKTaskName } from '../types/commonType';
+import { TaskResultEntity, TaskResult } from '../types/taskResult';
+import { TaskResultDaoImpl } from '../utils/db/taskResultDaoImpl';
+import { TaskResultDao } from '../utils/db/taskResultDao';
+import { CodeGeneration } from '../types/codeGeneration';
+import { CodeGenerationDaoImpl } from '../utils/db/codeGenerationDaoImpl';
+import { CodeGenerationDao } from '../utils/db/codeGenerationDao';
+
+export async function sendDB(
```

Comment:
Why define one function to send data to two DB (sdkGeneration, TaskResult)? It should be separated.

Reply:
done